neural_lam/models/ar_model.py

```diff
@@ -593,3 +593,11 @@ def on_load_checkpoint(self, checkpoint):
                 )
                 loaded_state_dict[new_key] = loaded_state_dict[old_key]
                 del loaded_state_dict[old_key]
+        if not self.restore_opt:
+            optimizers, lr_schedulers = self.configure_optimizers()
```

Comment:
`configure_optimizers` does not return two arguments (only the optimizer itself)

Reply:
You are right, I think this PR only works if the checkpointing is handled by wandb with `log_model=True` as seen in #16. I will remove the changes to checkpointing from #16 and make sure that all relevant changes for checkpointing and restoration are present in this PR.